src/app/logic/calculatePoints.js

```diff
@@ -0,0 +1,34 @@
+// eslint-disable-next-line import/extensions
+import { DIFFICULTY_LEVELS } from '../data/consts.js';
+
+export function calculatePoints(
+  correctAnswersScore,
+  isAnswerCorrect,
+  difficultyLevelsProperty,
+) {
+  if (
+    DIFFICULTY_LEVELS[difficultyLevelsProperty].calculatingPoints ===
+    DIFFICULTY_LEVELS.medium.calculatingPoints
+  ) {
+    let newCorrectAnswersScore = correctAnswersScore;
+    if (isAnswerCorrect) {
+      newCorrectAnswersScore = correctAnswersScore + 1;
+    }
+    return newCorrectAnswersScore;
```

Comment:
Perhaps you could think of shortening this part of the code. Maybe you can make use of this notation: `isAnswerCorrect? correctAnswerScore + 1 : correctAnswerScore`

Reply:
ESLint doesn't like here ternary operator

Reply:
![ternary](https://user-images.githubusercontent.com/92270179/147691503-f5db24fb-577e-4888-8173-220b9c5ca47b.jpg)

Reply:
`  let newScore;
  if (isMediumMode) {
    newScore = isAnswerCorrect ? correctAnswersScore + 1 : correctAnswersScore;
  }
  if (isHardMode) {
    newScore = isAnswerCorrect
      ? correctAnswersScore + 1
      : correctAnswersScore - 1;
  }
  return newScore;`

Reply:
that works! great!